build_runner/README.md

```diff
@@ -74,6 +74,7 @@ The available commands are `build`, `watch`, `serve`, and `test`.
 - `test`: Runs a single build, creates a merged output directory, and then runs
   `dart run test --precompiled <merged-output-dir>`. See below for instructions
   on passing custom args to the test command.
+- `build-filter`: Build filters allow you to choose explicitly which files to build instead of building entire directories.
```

Comment:
We don't document the individual options here - this should appear in `--help`?

There may also be an appropriate section in this doc to discuss it though, the main docs for this feature are at https://github.com/dart-lang/build/blob/master/docs/partial_builds.md

Reply:
Ah yeah it should go under Command Line Options (https://github.com/dart-lang/build/blob/9e405b30b712a55b202749664c3989074a8a6272/build_runner/README.md#command-line-options)

Reply:
Okay, sounds like this PR can be closed then. Thanks for the feedback.

Reply:
Or at the very least, we should include a link to partial_builds.md from the [main page](https://github.com/dart-lang/build). As it seems like a hidden feature of build_runner

Reply:
I don't think we need to close this - we should just list it under the command line options section (in the main readme)

Reply:
But I can also take this over if you would like

Reply:
Actually I think I can just push a change to this PR so I will try that

Reply:
ah that isn't enabled for your branch, I will open a new PR